### PR TITLE
Add member relations content

### PR DESCRIPTION
**Pull request by lscheske**
Add content to the activity "member relations", that currently reflects our membership process:

1. Membership growth
2. Member onboarding
3. Membership

Solves #616 
Solves #618

-----
[View rendered activities/member-relations/index.md](https://github.com/publiccodenet/about/blob/add-member-relations-content/activities/member-relations/index.md)

### CODE REVIEW

File: activities/member-relations/index.md [hunk 1]
```diff
@@ -4,4 +4,74 @@ type: Index
...
+
+During member growth we connect with and support wholly publically-owned organizations in their path towards membership.
+
+In this phase, we aim to:
+
+* build trust in our organization and our processes
+* determine focal and contact points within the prospective member organization
+* gain insight into the member's organizational structures and how codebase stewardship can benefit them
+
+The following are examples of activities we may conduct during member acquisition:
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:23]: Nested bullets are hard to read. Alternative suggestion:
```
We conduct various activities during member acquisition, like meetings to exchange knowledge and support with contracting. We can also offer workshops to raise awareness about:

  * codebase stewardship
  * public code
  * the [Standard for Public Code](https://standard.publiccode.net/)
  * our organizational model

To make it easier for the potential member, we have regular calls to answer any questions and ensure continuous, open communication.
```

File: activities/member-relations/index.md [hunk 2]
```diff
@@ -4,4 +4,74 @@ type: Index
...
+  * The Standard for Public Code
+  * Our organizational model
+* Hold meetings to exchange knowledge and support with contracting
+* Have regular calls to answer any questions and ensure continuous, open communication
+
+Member growth is finalized when the public organization signs a [membership contract](../membership-growth/membership-contract.md).
+
+## 2. Member onboarding
+
+During member onboarding we help new members get accustomed to our processes, and support their public code projects.
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:37]:
```suggestion
During member onboarding we help new members learn our processes, and support their public code projects.
```

File: activities/member-relations/index.md [hunk 3]
```diff
@@ -4,4 +4,74 @@ type: Index
...
+* support members with their public code projects and codebases
+* introduce members to our network and codebase communities
+* prepare members for their first general assembly
+
+The following are examples of activities we may conduct during member onboarding:
+
+* Develop a membership plan to identify members' support needs, and a timeline of milestones
+* Assess codebases for codebase stewardship
+* Conduct trainings on various topics surrounding public code
+* Conduct a [membership workshop](../membership-growth/founding-membership-workshop.md) to:
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:50]: This workshop needs updating to no longer say 'founding member'.

File: activities/member-relations/index.md [hunk 4]
```diff
@@ -4,4 +4,74 @@ type: Index
...
+
+During member onboarding we help new members get accustomed to our processes, and support their public code projects.
+
+In this phase, we aim to:
+
+* support members with their public code projects and codebases
+* introduce members to our network and codebase communities
+* prepare members for their first general assembly
+
+The following are examples of activities we may conduct during member onboarding:
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:45]: Another alternative to nested bullets.

```
During member onboarding, we usually conduct a [membership workshop](../membership-growth/founding-membership-workshop.md) to:
  * identify public code projects and codebases to collaborate on
  * set expectations for membership

We also do other activities that seem useful, for example:

* developing a membership plan to identify the member's support needs, and a timeline of milestones
* conducting trainings on various topics surrounding public code
* assessing codebases for codebase stewardship
```

File: activities/member-relations/index.md [hunk 5]
```diff
@@ -4,4 +4,74 @@ type: Index
...
+
+* Fully support members with all elements of codebase stewardship
+* Build capacity on various topics surrounding public code
+* Empower members to take ownership of steering our processes, governance, and finances
+* Maintain healthy communications amongst members and the Foundation for Public Code team
+
+The following are examples of activities we may conduct with our members:
+
+* Codebase stewardship
+* Build capacity on various topics surrounding public code
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:68]: Strange verb tense for the lead-in sentence. Alternative:
```
* Building capacity on various topics surrounding public code
* Brokering collaborative solutions within member community
* Ensuring good communications and active engagement within member community
```

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
This looks like really useful content for potential members, plus our staff. I've made some editing suggestions. There's also no need to capitalize the first letter of a bullet where the bullet follows on gramatically from lead in sentence - essentially, each bullet then functions as an alternative ending to the lead in sentence.

File: activities/member-relations/index.md (hunk 3 above)
Comment (lscheske) [reply activities/member-relations/index.md:50]: Will do!

File: activities/member-relations/index.md [hunk 6]
```diff
@@ -4,4 +4,74 @@ type: Index
...
 
+The Foundation for Public Code is a member-owned association. Only wholly publicly owned organizations may become members. Members participate in the steering and governance of the association by voting at the general assembly. They also participate by financing the association through yearly dues.
+
 We support members in getting the most from their membership and [codebase stewardship](../codebase-stewardship), as well as building and maintaining relationships with and between members.
+
+The process towards full membership is roughly divided into three phases, noting that every organization we speak to is unique. This means that individual membership  processes may differ between the organizations we work with.
+
+## 1. Membership growth
+
+During member growth we connect with and support wholly publically-owned organizations in their path towards membership.
```
[the remark this answers is not in the document]
Comment (lscheske) [reply activities/member-relations/index.md]: I went with membership growth for now as this is what we have been using so far and currently have a common understanding about. I personally like your suggestion of using "cultivation" and I think we should consider using this in the future!

**Review by bvhme — DISMISSED** [review]

File: activities/member-relations/index.md [hunk 7]
```diff
@@ -4,4 +4,76 @@ type: Index
 
 # Member relations
 
+The Foundation for Public Code is a member-owned association. Only wholly publicly owned organizations may become members. Members participate in the steering and governance of the association by voting at the general assembly. They also participate by financing the association through yearly dues.
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:7]:
```suggestion
The Foundation for Public Code is a member-owned association. Only wholly publicly-owned organizations may become members. Members participate in the steering and governance of the association by voting at the general assembly. They also participate by financing the association through yearly dues.
```

File: activities/member-relations/index.md [hunk 8]
```diff
@@ -4,4 +4,76 @@ type: Index
...
+
 We support members in getting the most from their membership and [codebase stewardship](../codebase-stewardship), as well as building and maintaining relationships with and between members.
+
+The process towards full membership is roughly divided into three phases, noting that every organization we speak to is unique. This means that individual membership  processes may differ between the organizations we work with.
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:11]:
```suggestion
The process towards full membership is roughly divided into three phases, noting that every organization we speak to is unique. This means that individual membership processes may differ between the organizations we work with.
```

File: activities/member-relations/index.md [hunk 9]
```diff
@@ -4,4 +4,76 @@ type: Index
...
+During membership growth we connect with and support wholly publicly-owned organizations in their path towards membership.
+
+In this phase, we aim to:
+
+* build trust in our organization and our processes
+* determine focal and contact points within the prospective member organization
+* gain insight into the member's organizational structures and how codebase stewardship can benefit them
+
+We conduct various activities during membership growth, like meetings to exchange knowledge and support with contracting. We can also offer workshops to raise awareness about:
+
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:26]: The bullets below are double indented, which irks Travis.

File: activities/member-relations/index.md (hunk 2 above)
Comment (ElenaFdR) [reply activities/member-relations/index.md:38]: "Get accustomed to" sounds passive, slow and perhaps a bit unpleasant - like growing a callous where a too-tight shoe rubs your foot. Is there a more active/proactive/positive description of this phase?

File: activities/member-relations/index.md [hunk 10]
```diff
@@ -4,4 +4,76 @@ type: Index
...
+During member onboarding we help new members get accustomed to our processes, and support their public code projects.
+
+In this phase, we aim to:
+
+* support members with their public code projects and codebases
+* introduce members to our network and codebase communities
+* prepare members for their first general assembly
+
+During member onboarding, we usually conduct a [membership workshop](../membership-growth/founding-membership-workshop.md) to:
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:46]: The bullets below are double indented, which irks Travis.

File: activities/member-relations/index.md [hunk 11]
```diff
@@ -4,4 +4,76 @@ type: Index
...
+* Building capacity on various topics surrounding public code
+* Brokering collaborative solutions within member community
+* Ensuring good communications and active engagement within member community
+
+## Further Reading
+
+* The [membership growth](../membership-growth) process
+* Our [mission, founding principles and goals](../../organization/mission.md)
+* Overview of our [activities](../../activities/index.md)
+* [Codebase stewardship](../codebase-stewardship/)
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:79]: Travis would like another blank line after this list.

File: activities/member-relations/index.md [hunk 12]
```diff
@@ -4,4 +4,76 @@ type: Index
...
+* Fully support members with all elements of codebase stewardship
+* Build capacity on various topics surrounding public code
+* Empower members to take ownership of steering our processes, governance, and finances
+* Maintain healthy communications amongst members and the Foundation for Public Code team
+
+The following are examples of activities we may conduct with our members:
+
+* Codebase stewardship
+* Building capacity on various topics surrounding public code
+* Brokering collaborative solutions within member community
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:71]: This looks gramatically incorrect - does it mean between members (our member community) or is it missing an article and a possessive apostrophe (a member's community)?

File: activities/member-relations/index.md [hunk 13]
```diff
@@ -4,4 +4,76 @@ type: Index
...
+* conducting trainings on various topics surrounding public code
+* assessing codebases for codebase stewardship
+
+## 3. Membership
+
+During membership, members become fully involved in governing the Foundation for Public Code and build mature public code projects in their organizations.
+
+In this phase, we aim to:
+
+* Fully support members with all elements of codebase stewardship
```
Comment (ElenaFdR) [inline activities/member-relations/index.md:62]: The bullets from here down start with capital letters, which is inconsistent with previous bulleted lists on the page

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
Hello! I believe I didn't rereview this because it still broke Travis; sorry for not actually telling you that. It looks so close to being done! And will still be useful and helpful.

**Review by lscheske — COMMENTED** [review]
Addressed all comments, checks now pass.

**Review by lscheske — COMMENTED** [review]
committed change in line 38 (replace "get accustomed to" with "learn"

**Review by ElenaFdR — APPROVED** [review]
This looks great - set it free!

**Pull request merged**